modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,29 @@
+[id="rhel-images-aws_{context}"]
+= Listing latest available RHEL images on AWS
+
```

Comment:
Since this is a procedure module, I'd recommend formatting it with `.Procedure` and `.Prerequisites` (if necessary) headings. For example:

```
[id="rhel-images-aws_{context}"]
= Listing latest available RHEL images on AWS

You can use the AWS CLI to blah blah blah.

.Prerequisites

* You have installed the AWS CLI.
* etc...

.Procedure

* Do this command:
...
```